### DISCUSSION TITLE
Possible to have multiple lines/splitters to seperate things on a single line?

### DISCUSSION THREAD

**Discussion by KingKDot** [zauberzeug/nicegui#3352] [Q&A] [answered]
### Question

Right now I have the following code

```py
        with ui.splitter() as splitter:
            with splitter.before:
                for key, label in list(checkbox_options.items())[:4]:
                    checkbox_values[key] = ui.checkbox(label).classes("w-full")
            with splitter.after:
                for key, label in list(checkbox_options.items())[4:]:
                    checkbox_values[key] = ui.checkbox(label).classes("w-full")
```

but I want to know if it's possible to make it so there is more than just before and after or if there is a different tool I can use to achieve this. Thank you.

**Comment by python-and-novella** [comment, marked as answer]
splitter is container, separator is simple control.
```python3
from nicegui import ui

with ui.row():
    for i in range(9):
        with ui.column():
            for k in range(i+1):
                ui.label(str(k))
        ui.separator().props('vertical')
    else:
        with ui.column():
            for k in range(i+1+1):
                ui.label(str(k))

ui.run()
```

**Reply by falkoschindler** [reply]
I assume @KDot227 is asking for draggable separators.

**Reply by python-and-novella** [reply]
splitter can be embedded in splitter. https://quasar.dev/vue-components/splitter#embedded
But it is a little complex.
Maybe draft pic and more details can help us to understand clearly. @KDot227

**Reply by KingKDot** [reply]
> I assume @KDot227 is asking for draggable separators.

They don't have to be drag-able I just want a nice looking separation.

**Comment by falkoschindler** [comment]
The current implementation of `ui.splitter`, which is based on [QSplitter](https://quasar.dev/vue-components/splitter/), doesn't support multiple splits. But maybe someone can come up with a workaround?
